pkg/ibmcsidriver/controller_helper.go

```diff
@@ -303,8 +303,6 @@ func isValidCapacityIOPS(size int, iops int, profile string) (bool, error) {
 
 	if profile == DP2Profile {
 		capacityIopsRanges = dp2CapacityIopsRanges
-	} else if profile == CustomProfile {
-		capacityIopsRanges = customCapacityIopsRanges
```

Comment:
@arahamad lets remove this map also customCapacityIopsRanges

Reply:
done!